### DISCUSSION TITLE
part-ii/2008-64/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#90] [Question comments]
Part II 2008 Graph Theory: 1.II .17F 

https://questions.tripos.org/part-ii/2008-64/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

The first part is bookwork.

i) False. $K_{3,3}$ is 2-colourable but is not planar.

ii) False. Take a $K_5$ and turn it into a subdivision by replacing one edge with a path of length 1000 (so we have added 999 vertices). This new graph has 1004 vertices and 1009 edges, so the average degree is $\frac{2018}{1004} < 2.01$, but this graph is not planar since $K_5$ is not planar (this inequality is not obvious without a calculator, but we can just add more vertices to the path). 

ii) True. We use Kuratowski's theorem. Any subdivision of $K_5$ or $K_{3,3}$ has average degree greater than 2 (since every vertex has degree at least 2), and adding edges to a subdivision will only increase the average degree. Meanwhile if we add more vertices, each vertex we add will be adjacent to at least one vertex in the graph since we require the graph to be connected, and adding a vertex and an edge keeps the average degree the same.

</details>
